src/Microsoft.Identity.Client/Instance/AadAuthority.cs

```diff
@@ -38,19 +37,6 @@ internal static bool IsInTrustedHostList(string host)
         {
         }
 
-        internal override async Task UpdateCanonicalAuthorityAsync(
```

Comment:
This logic also doesn't need to scoped to AAD authority only. For B2C and ADFS, the initial authority is always the same as metadata.PreferredNetwork. Using this makes the code easier to navigate.